Formula/mpv.rb

```diff
@@ -64,12 +69,30 @@ def install
     system "meson", "compile", "-C", "build", "--verbose"
     system "meson", "install", "-C", "build"
 
+    if OS.mac?
+      # `pkg-config --libs mpv` includes libarchive, but that package is
+      # keg-only so it needs to look for the pkgconfig file in libarchive's opt
+      # path.
+      libarchive = Formula["libarchive"].opt_prefix
+      inreplace lib/"pkgconfig/mpv.pc" do |s|
+        s.gsub!(/^Requires\.private:(.*)\blibarchive\b(.*?)(,.*)?$/,
+                "Requires.private:\\1#{libarchive}/lib/pkgconfig/libarchive.pc\\3")
+      end
+    end
+
     bash_completion.install "etc/mpv.bash-completion" => "mpv"
     zsh_completion.install "etc/_mpv.zsh" => "_mpv"
   end
 
   test do
-    system bin/"mpv", "--ao=null", "--vo=null", test_fixtures("test.wav")
+    test_media = test_fixtures("test.wav")
+
+    system bin/"mpv", "--ao=null", "--vo=null", test_media
     assert_match "vapoursynth", shell_output(bin/"mpv --vf=help")
+
+    resource("homebrew-simple.c").stage testpath
+    pkg_config_flags = shell_output("pkg-config --libs --cflags mpv").chomp.split
```

Comment:
Just doing
```ruby
system "pkg-config", "mpv"
```
would've been enough, because that checks that `pkg-config` can parse `mpv.pc`. You also need
```ruby
depends_on "pkg-config" => [:build, :test]
```

Reply:
Yea, but well, since we're already here...

Reply:
Actually, maybe not. This requires user interaction, but that may be too much.